WooCommerce/WooCommerceTests/Notifications/PushNotificationTests.swift

```diff
@@ -0,0 +1,52 @@
+//import Combine
+import Experiments
+import XCTest
+import Yosemite
+@testable import WooCommerce
+
+final class PushNotificationTests: XCTestCase {
+    private let userInfo: [AnyHashable: Any] = [
```

Comment:
Used that payload from testing here for test so we use it for this test and if needed in the future we can use it to copy/paste it if we need to trigger some local notification :)